Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -372,6 +378,7 @@ YUI.add('ez-locationviewviewservice', function (Y) {
                 location: this.get('location'),
                 path: this.get('path'),
                 config: this.get('config'),
+                languageCode: this.get('languageCode'),
```

Comment:
that's incorrect, we should send to the view the languageCode coming from the URI so, that should be:
```js
languageCode: this.get('request').params.languageCode,
```